datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -78,10 +78,25 @@ export const EditOwnersModal = ({
     const renderSearchResult = (entity: Entity) => {
         const avatarUrl =
             (entity.type === EntityType.CorpUser && (entity as CorpUser).editableProperties?.pictureLink) || undefined;
+        const isCorpUser = (entity.type as EntityType) === EntityType.CorpUser;
+        const displayCorpUserDepartmentName =
+            (entity.type === EntityType.CorpUser && (entity as CorpUser).properties?.departmentName) || '';
+        const displayCorpUserUserName = (entity.type === EntityType.CorpUser && (entity as CorpUser).username) || '';
```

Comment:
I'd prefer to see this name UNDERNEATH the primary name of the user in gray: 

<img width="413" alt="Screenshot 2024-11-25 at 10 05 14 AM" src="https://github.com/user-attachments/assets/3ec21ae0-af6f-4b96-a017-f5f147129d53">

Can we refactor to move towards this design?

Reply:
Addressed